src/login/styles/_inputs.scss

```diff
@@ -93,15 +97,15 @@ input::placeholder {
   font-size: 1.5rem;
 }
 
-// .form-control:focus,
+.form-control:focus,
 input:focus {
   outline: 0;
   border-bottom: solid 2px $black;
   box-shadow: 0 0 0 0.2rem $mid-gray;
 }
 
 .form-control.is-valid:focus {
-  border-color: $black;
+  border-color: #28a745;
```

Comment:
Should this colour be turned into a variable?

Reply:
this has been updated after merge from latest mater